packages/core/lib/command.js

```diff
@@ -76,6 +76,12 @@ class Command {
   async run(inputStrings, options) {
     const result = this.getCommand(inputStrings, options.noAliases);
 
+    try {
```

Comment:
I built this to fail silently but perhaps we should consider adding some sort of a warning message here so that if there are issues people can report them?

Reply:
I think it would be nice to console out a message to the effect of "unable to enable user-level config file. please let the Truffle team know" or something?

Reply:
definitely not a dealbreaker either way though

Reply:
Can you add a `debug` module output at least? It would be useful to turn on to track issues?